test/helpers/gas-usage.ts

```diff
@@ -13,21 +13,29 @@ export enum GasGroup {
   BULK_TRANSFER_VALIDATOR,
   BULK_TRANSFER_VALIDATOR_NON_EXISTING_POD,
   LIQUIDATE_VALIDATOR,
+  DEPOSIT,
+  WITHDRAW,
+  REGISTER_POD,
+  REACTIVATE_POD,
 }
 
 const MAX_GAS_PER_GROUP: any = {
   [GasGroup.REGISTER_OPERATOR]: 105000,
   [GasGroup.REMOVE_OPERATOR]: 45000,
   [GasGroup.REGISTER_VALIDATOR_EXISTING_POD]: 202000,
   [GasGroup.REGISTER_VALIDATOR_EXISTING_CLUSTER]: 220000,
-  [GasGroup.REGISTER_VALIDATOR_NEW_STATE]: 333000, // 313000
+  [GasGroup.REGISTER_VALIDATOR_NEW_STATE]: 319000, // 313000
   [GasGroup.REMOVE_VALIDATOR]: 120000,
   [GasGroup.TRANSFER_VALIDATOR_NEW_CLUSTER]: 400000,
   [GasGroup.TRANSFER_VALIDATOR]: 260000,
   [GasGroup.TRANSFER_VALIDATOR_NON_EXISTING_POD]: 290000,
   [GasGroup.BULK_TRANSFER_VALIDATOR]: 362000,
   [GasGroup.BULK_TRANSFER_VALIDATOR_NON_EXISTING_POD]: 379000,
-  [GasGroup.LIQUIDATE_VALIDATOR]: 101000,
+  [GasGroup.LIQUIDATE_VALIDATOR]: 120000,
+  [GasGroup.DEPOSIT]: 84000,
```

Comment:
@vadiminc @andrew-blox This new gases are too much
please revisit

Reply:
@andrew-blox @lior-blox i change deposit flow to work with pod and it saved us 5k gas. So now it's 79k gas usage
part of it:
- 20k - token transfer
- 2k - emit event,
the rest is the logic.